package.json

```diff
@@ -4,8 +4,16 @@
   "description": "A ponyfill (doesn't overwrite the native fetch) for the Fetch specification https://fetch.spec.whatwg.org.",
   "main": "fetch-node.js",
   "browser": "build/fetch-browser.js",
+  "config": {
+    "web_port": "8088"
+  },
   "scripts": {
     "test": "mocha tests/fetch-node.spec.js",
+    "browser_test_helper": "(sleep 1 && opener http://localhost:$npm_package_config_web_port/tests/fetch-browser.spec.html) && http-server -p $npm_package_config_web_port",
+    "pretest:browserify": "npm run build && browserify tests/fetch-browser.spec.js --outfile build/bundle.js",
```

Comment:
Nitpicking a bit... would you mind renaming the bundle to something like `test-bundle.js` or similar?